sdk/cosmosdb/arm-cosmosdb/samples-dev/cassandraClustersDeleteSample.ts

```diff
@@ -18,12 +18,10 @@ dotenv.config();
  * This sample demonstrates how to Deletes a managed Cassandra cluster.
  *
  * @summary Deletes a managed Cassandra cluster.
- * x-ms-original-file: specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2023-04-15/examples/CosmosDBManagedCassandraClusterDelete.json
+ * x-ms-original-file: specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2023-03-15-preview/examples/CosmosDBManagedCassandraClusterDelete.json
  */
 async function cosmosDbManagedCassandraClusterDelete() {
-  const subscriptionId =
-    process.env["COSMOSDB_SUBSCRIPTION_ID"] ||
-    "00000000-0000-0000-0000-000000000000";
+  const subscriptionId = process.env["COSMOSDB_SUBSCRIPTION_ID"] || "subid";
```

Comment:
why?

Reply:
compare with current  api version and required api version, this change because in required version , it changes to subid in swagger example:
![image](https://github.com/Azure/azure-sdk-for-js/assets/98569699/8fc1439a-b94c-43fe-aa9a-99fcb60072e4)